src/content/reference/developer-tools/cli.md

```diff
@@ -557,6 +557,65 @@ Example usage:
 | `particle bundle myApp.bin --assets /path/ --saveTo myApp.zip` | Creates a bundle of application binary and assets, and saves it to the myApp.zip file |
 | `particle bundle myApp.bin --saveTo myApp.zip` | Creates a bundle of application binary and assets as specified in the assetOtaDir if available, and saves the bundle to the myApp.zip file |
 
+## particle device-protection
+
+[Device protection](/scaling/best-practices/device-protection/) is an enterprise feature of Device OS 6.x to provide additional 
+security for devices to prevent local flashing by USB or SWD/JTAG, USB serial access, and other features. 
+
+Devices cannot be removed from device protection using the Particle CLI, but certain features such as USB
+serial access can be temporarily disabled.
+
+You must be logged into a your Particle account in the CLI and have appropriate access to the product the protected device
```

Comment:
@rickkas7 https://www.notion.so/particle/Device-Protection-Customer-Messaging-d08116e4c2a5480da1846aaee6fb234d

Device Protection is title case because it is a noun. So is Protected Device. The above doc has more info.

Reply:
Fixed